edx_sga/static/js/src/edx_sga.js

```diff
@@ -376,8 +381,47 @@ function StaffGradedAssignmentXBlock(runtime, element) {
                   );
                 });
             }
+            $("#lean_overlay").on("click", function (_){
+            if (currentIFrameHeight && !gradePopUpIsOpen) {
+              sendResizeMessage(currentIFrameHeight)
+              currentIFrameHeight = null
+            }
+            })
         });
 
+        function updateIframe(){
+          if(isInIframe && !currentIFrameHeight){
+            currentIFrameHeight = $("body").height()
+            addMaxHeightInIframe()
+            if(currentIFrameHeight < 600){
+              sendResizeMessage(currentIFrameHeight + 600)
+            }
+          }
+        }
+
+        function isInIframe(){
+          return window.parent !== window
+        }
+
+        function addMaxHeightInIframe(){
+            $(".grade-submission").css("max-height", "600px")
```

Comment:
don't you think, it should rather be applied on iframe so that it could cover all the iframe size rather on a particular iframe with `.grade-submission`?

Like we have Staff debug, or other iframes too. So these other iframes would remain scattered and distorted. Let me know what's your opinion on that.

Reply:
And if we go to choose to apply it on `iframe`, it should be a part of `.scss/css`. Just try to load another popup like Staff Debug / Debug info with staff user

Reply:
The iframe tag is out of scope from the SGA block and so we cant access it using it using jQuery or vanilla JS.
We could however move the style changes from edx-sga repo to learner-MFE as that controls the various Iframes rendered here.

Reply:
and does it have an effect the legacy view?

Reply:
No